test/oph/ehoks/db/dynamodb_test.clj

```diff
@@ -36,6 +36,11 @@
                 :osaamisen-hankkimisen-tarve true
                 :sahkoposti "irma.isomerkki@esimerkki.com"})
 
+(deftest test-sync-tpo-nippu-herate!
+  (with-redefs [koski/get-opiskeluoikeus-info-raw
+                koski-test/mock-get-opiskeluoikeus-raw]
+    nil))
```

Comment:
Olikohan tässä kohtaa jätetty testi tarkoituksella pois / myöhemmin kirjoitettavaksi?

Reply:
Tää oli alunperin tarkoitettu myöhemmin kirjoitettavaksi, mutta tää itseasiassa vaan unohtui multa koodiin, koska ei ollut lopulta enää tarvetta tehdä yksikkötestiä `sync-tpo-nippu-herate!`-funktiolle :sweat_smile: Tuon toiminnallisuus tulee testattua enemmän tai vähemmän testissä `test-create-and-save-arvo-vastaajatunnus-for-all-needed!`

Reply:
Odotan loputkin katselmointikommentit ja siivoan tuon sen jälkeen pois.

Reply:
Merkkaan takaisin "unresolved"-tilaan, toimii itselle muistutuksena että pitää vielä korjata.